modules/programs/numbat.nix

```diff
@@ -38,11 +38,27 @@ in
         <https://numbat.dev/doc/cli-customization.html#configuration> for options.
       '';
     };
+
+    initFile = lib.mkOption {
+      type = lib.types.nullOr (lib.types.either lib.types.lines lib.types.path);
```

Comment:
I find it weird (it might exist elsewhere) to mix path and content though not blocking.

Reply:
Personally, I like the flexibility of options that support both. I've tried to implement it wherever I can. We also have a special lib type for supporting directories (for more complex configurations) that I use in sketchybar: https://github.com/nix-community/home-manager/blob/master/modules/programs/sketchybar.nix#L48-L48